File: app/src/main/java/io/mgba/Utils/DisplayUtils.java
package io.mgba.Utils;

import android.app.Activity;
import android.content.Context;
import android.util.DisplayMetrics;

public class DisplayUtils {

    public static float getDisplayHeight(Context context){
        return prepareDisplayMetrics(context).heightPixels;
    }

    public static float getDisplayWidth(Context context){
        return prepareDisplayMetrics(context).widthPixels;
    }

    private static DisplayMetrics prepareDisplayMetrics(Context context){
        DisplayMetrics displayMetrics = new DisplayMetrics();
        ((Activity)context).getWindowManager().getDefaultDisplay().getMetrics(displayMetrics);

        return displayMetrics;
    }
}

File: app/src/main/java/io/mgba/utilities/DisplayUtils.kt
package io.mgba.utilities

import android.app.Activity
import android.content.Context
import android.util.DisplayMetrics

object DisplayUtils {

    fun getDisplayHeight(context: Context): Float {
        return prepareDisplayMetrics(context).heightPixels.toFloat()
    }

    fun getDisplayWidth(context: Context): Float {
        return prepareDisplayMetrics(context).widthPixels.toFloat()
    }

    private fun prepareDisplayMetrics(context: Context): DisplayMetrics {
        val displayMetrics = DisplayMetrics()
        (context as Activity).windowManager.defaultDisplay.getMetrics(displayMetrics)

        return displayMetrics
    }
}
